controllers/imagecollector/imagecollector_controller.go

```diff
@@ -198,6 +250,20 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 			}
 			return ctrl.Result{}, nil
 		}
+
+		// if scan is disabled, create/update imagelist to prune since collector job has finished
+		if scanDisabled {
```

Comment:
if you really want the clarity that this name provides you can write a simple function
```
func scanDisabled() bool {
    return *scannerImage == ""
}
```